mlbstatsapi/mlbapi.py

```diff
@@ -186,4 +194,31 @@ def get_division_id(self, divisionName) -> List[Division]:
         for division in mlbdata.data['divisions']:
             if division['name'].lower() == divisionName.lower(): # Match division name
                 divisionIds.append(division['id']) # add to list
-        return divisionIds
\ No newline at end of file
+        return divisionIds
+
+    def get_attendance(self, teamId=None, leagueId=None, season=None, date=None,
```

Comment:
Can you please explain this function to someone who isn't experienced with the game endpoint?